price-feeder/oracle/provider/binance.go

```diff
@@ -102,6 +100,26 @@ func NewBinanceProvider(ctx context.Context, logger zerolog.Logger, pairs ...typ
 	return provider, nil
 }
 
+// SubscribeTickers subscribe all currency pairs into ticker and candle channels.
+func (p *BinanceProvider) SubscribeTickers(cps ...types.CurrencyPair) error {
+	pairs := make([]string, len(cps)*2)
+
+	iterator := 0
+	for _, cp := range cps {
+		pairs[iterator] = currencyPairToBinanceTickerPair(cp)
+		iterator++
+		pairs[iterator] = currencyPairToBinanceCandlePair(cp)
+		iterator++
+	}
```

Comment:
is `cps` guaranteed to be an alternating list of Ticker + Candle?

Reply:
for the moment, yes 
- maybe after [tvwap](https://github.com/umee-network/umee/pull/597) implemented we could only use candles when available

Reply:
oooo I didn't realize this. I'm not really a fan of that approach. It's too brittle IMO. Could we remove that assumption? Maybe just pass two args in instead?

Reply:
After [tvwap](https://github.com/umee-network/umee/pull/601) gets merged can we put these back as separate functions? 

Candle & Ticker pairs should generally be considered separate, especially since candles can not always be used immediately (you can't calculate tvwap if the only candle you have is one that closed at that exact moment). Would make this less brittle too.

Reply:
Well, these pairs are used only to subscribe to those channels. We could have 2 lists of subscribed pairs and separate them one for each type (Ticker, Candle). 

The logic to subscribe in both of those channels was the same. So, in my idea made sense to only iterate through `cps ...types.CurrencyPair` once 

@adamewozniak @toteki @alexanderbez

Reply:
Would rather iterate twice and keep the code clean and legible 🙏 

These aren't big lists anyway